grails-app/views/report/showOnOrderReport.gsp

```diff
@@ -158,9 +158,9 @@
                 } );
             },
             "fnRowCallback": function( nRow, aData ) {
-                $('td:eq(1)', nRow).text(aData?.translatedProductName ?? aData?.productName);
+                $('td:eq(1)', nRow).text(aData?.displayName ?? aData?.productName);
```

Comment:
doesn't my comment from above refer to this? I can't see you having `displayName` in the json